examples/webgpu_rtt.html

```diff
@@ -80,7 +90,9 @@
 				const screenFXNode = uniform( mouse );
 
 				const materialFX = new THREE.MeshBasicNodeMaterial();
-				materialFX.colorNode = hue( saturation( texture( renderTarget.texture ).rgb, screenFXNode.x.oneMinus() ), screenFXNode.y );
+
+				const scenePassTexture = texture( renderTarget.texture ).toInspector( 'Scene Pass' );
```

Comment:
One thing I have noticed: The preview of the scene pass is rendered correctly in the `Viewer` tab but in the `Performance` tab I see this:

<img width="337" height="41" alt="image" src="https://github.com/user-attachments/assets/4da98c34-e980-4924-a442-b9e33b3ac947" />

The scene pass entry is remove after a few seconds? Is that a bug or do I miss something in the setup?

Reply:
This is the `Viewer` preview render call; we could name it Viewer instead of Inspector for clarity.

There are calls that are made only once, and others that are removed. These remain persistent for inspection purposes for 5 seconds, which are counted in asterisks. If the call is very quick, such as a computation or rendering performed in only one frame, the user won't be able to inspect it.

Reply:
I added `quadMesh.name = 'Post-Processing'` to better show that it would be post-processing in the performance tab and I made the renaming mentioned above.